app/javascript/components/venue/VenueEdit.jsx

```diff
@@ -0,0 +1,96 @@
+import React, { Component } from 'react';
+import Axios from 'axios';
+
+export default class VenueEdit extends Component {
+  constructor(props) {
+    super(props);
+    
+    this.state = { venue: { address: '', description: '', people_capacity: '' } };
+ 
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange = (venue) => {
+    this.setState({
+      [venue.target.name]: venue.target.value
+    });
+  }
+
+  handleSubmit = (event) => {
+  event.preventDefault();
+  const { address, description, people_capacity } = this.state;
+  const csrf = document.querySelector("meta[name='csrf-token']").getAttribute("content");
+  Axios
+    .patch(
+      `/api/v1/venues/${this.props.match.params.id}`, 
+      {
+        address: address,
+        description: description,
+        people_capacity: people_capacity},
```

Comment:
Instead of 
```
{  address: address, 
   description: description,
   people_capacity: people_capacity},
```
try use ```this.state.venue```

Reply:
It is not working in that case, if you help me with that i will change it.